pkg/fission-cli/cmd/spec/spec.go

```diff
@@ -222,11 +223,20 @@ func SpecSave(resource interface{}, specFile string) error {
 		kind = typedres.TypeMeta.Kind
 		data, err = yaml.Marshal(typedres)
 	default:
+		if input.Bool(flagkey.SpecDry) {
+			return fmt.Errorf("can't display resource %#v spec", resource)
+		}
 		return fmt.Errorf("can't save resource %#v", resource)
 	}
 	if err != nil {
 		return errors.Wrap(err, "Couldn't marshal YAML")
 	}
+	if input.Bool(flagkey.SpecDry) {
+		console.Info(fmt.Sprintf("Generated Spec\n%v", (string(data))))
```

Comment:
People may want to do something like `fission create xxx --dry > a.yaml`.
So we can just print the crd YAML directly without adding prepending string.
And, `console.XXX` is to print the message for the user. So if you want to print the content of CRD, use `fmt.Printf(string(data))`.